### PR TITLE
Add support to get_by_name for interconnects

### PR DESCRIPTION
**Pull request by bsouza**

### CODE REVIEW

File: examples/interconnects.py [hunk 1]
```diff
@@ -146,10 +155,10 @@
     print(e.msg['message'])
 
 # Get name servers
-print("Get name servers that matches ID 929f265b-30dc-44bd-97f7-0942d56e9939")
+print("Get name servers that matches ID that matches tha specified ID")
```
Comment (marikrg) [inline examples/interconnects.py]: fix the typo: `tha specified`

Comment (marikrg) [inline examples/interconnects.py]: I believe "Get the named servers for the interconnect that matches the specified ID" sounds better.

File: hpOneView/resources/networking/interconnects.py [hunk 2]
```diff
@@ -148,6 +148,18 @@ def get_by(self, field, value):
         """
         return self._client.get_by(field, value)
 
+    def get_by_name(self, name):
+        """
+        Retrieve an Interconnect by his name
```
Comment (marikrg) [inline hpOneView/resources/networking/interconnects.py]: Replace `his name` by `its name`

File: examples/interconnects.py (hunk 1 above)
Comment (balestrinc) [inline examples/interconnects.py]: Done

File: hpOneView/resources/networking/interconnects.py (hunk 2 above)
Comment (balestrinc) [inline hpOneView/resources/networking/interconnects.py]: Done

**Pull request merged**